bazel/external/lightstep.BUILD

```diff
@@ -17,7 +17,7 @@ cc_library(
     hdrs = [":" + hdr for hdr in _PREFIX_HDRS],
     includes = ["_prefix/include"],
     visibility = ["//visibility:public"],
-    deps = ["@protobuf_bzl//:protobuf"],
+    deps = ["@com_google_protobuf//:protobuf"],
```

Comment:
Can those be `//external` ?

Reply:
We're moving away from bind now that we're switching to the Bazel dictated `@com_google_protobuf`.

Reply:
OK then load those from `@com_google_protobuf_cc` if it is C++ specific.